src/trans/gpu/internal/trgtol_mod.F90

```diff
@@ -124,7 +124,9 @@ SUBROUTINE TRGTOL(ALLOCATOR,HTRGTOL,PREEL_REAL,KF_FS,KF_GP,KF_UV_G,KF_SCALARS_G,
     USE BUFFERED_ALLOCATOR_MOD, ONLY: BUFFERED_ALLOCATOR, ASSIGN_PTR, GET_ALLOCATION
     USE OPENACC_EXT,            ONLY: EXT_ACC_ARR_DESC, EXT_ACC_PASS, EXT_ACC_CREATE, &
       &                               EXT_ACC_DELETE
```

Comment:
I'm surprised to see that OPENACC_EXT module is available without OpenACC, i.e. just with OpenMP offload.

Reply:
OPENACC_EXT is [one of ours](https://github.com/ecmwf-ifs/ectrans/blob/develop/src/trans/gpu/algor/ext_acc.F90) and should definitely be renamed if we are ever successful in adding support for OpenMP offload :)

Reply:
Oh OK, I did not realise.